conda

```diff
@@ -221,6 +223,7 @@ def _update_package(module, conda, installed, name):
     command = [
         conda,
         update,
```

Comment:
@jiffyclub You're right, that should be a string literal. I hadn't used that function myself yet.

I'm pushing a fix now...